src/App.jsx

```diff
@@ -23,23 +23,19 @@ export const alex = {
   isMarried: false,
 };
 
+const userData = [misha, olya, alex];
+
 export const App = () => (
   <div className="App">
-    <section className="Person">
-      <h2 className="Person__name">My name is Misha</h2>
-      <p className="Person__age">I am 37</p>
-      <p className="Person__partner">Natasha is my wife</p>
-    </section>
-
-    <section className="Person">
-      <h2 className="Person__name">My name is Olya</h2>
-      <p className="Person__partner">Maksym is my husband</p>
-    </section>
-
-    <section className="Person">
-      <h2 className="Person__name">My name is Alex</h2>
-      <p className="Person__age">I am 25</p>
-      <p className="Person__partner">I am not married</p>
-    </section>
+    {userData.map(user => (
+      <section className="Person">
+        <h2 className="Person__name">{`My name is ${user.name}`}</h2>
+        <p className="Person__partner">
+          {user.isMarried
+            ? `${user.partnerName} is my ${user.sex === 'm' ? 'wife' : 'husband'}`
+            : 'I am not married'}
+        </p>
+      </section>
+    ))}
```

Comment:
[CODE STYLE] - Readabily is everything. Format ternary operator operands correctly - move each operand to the separate line